department-of-veterans-affairs/vets-website
# add medallia styling as separate output

## GitHub Issue

[Host Medallia CSS #13287](https://github.com/department-of-veterans-affairs/va.gov-team/issues/13287) 

## Description

Remove hash from Medallia CSS file in staging environment

## Testing done

1. Run `yarn build:webpack:prod`
   - Build time with sourcemaps = ~50 minutes 😱 
   - Build time without sourcemaps = ~15 minutes 😐 
1. Run `ls -l build/vagovprod/generated | grep medal` to find the medallia CSS file
1. Confirm there is a hash in the filename (for example, `va-medallia-styles.a9a525ad4ac2474ed8b4-1601477867140.css`)
1. Run `yarn build:webpack:staging`
1. Run `ls -l build/vagovstaging/generated | grep medal` to find the medallia CSS file
1. Confirm there is no hash in the filename (for example, `va-medallia-styles.css`)

## Screenshots

![](https://user-images.githubusercontent.com/6130520/94706143-71924d80-0307-11eb-9b44-ef78f3f3b425.png)

## Acceptance criteria

## Definition of done
